Meshery-Adapters/traefik-meshery-adapter/step1-foreground.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+/root/shell.sh
```

Comment:
You need to think about `/root/shell.sh`, with `minikube` as image id we have to start Minikube using `/root/shell.sh`, which we don't want.
Now with `kubernetes-cluster-running` you need to identify if  `/root/shell.sh` is required.

Reply:
Alright, I'll have a look

Reply:
I checked out https://www.katacoda.com/warunicorn19/scenarios/traefik-meshery-adapter
Following needs work :

1 . On starting lab, it is still trying to do Minikube start via /root/shell.sh
2. After installing and starting meshery this happen.
<img width="977" alt="Screenshot 2021-09-01 at 8 28 24 AM" src="https://user-images.githubusercontent.com/41604114/131605210-f24288be-c284-43e6-ad08-d6844e89107a.png">

so, have to do `q`(quit)  and `y` (yes) to get 

<img width="856" alt="Screenshot 2021-09-01 at 8 39 54 AM" src="https://user-images.githubusercontent.com/41604114/131605503-ea559eb0-6733-4bef-80ec-9e9c5313d4e6.png">

3. Meshery dashboard shows nothing
<img width="976" alt="Screenshot 2021-09-01 at 8 30 02 AM" src="https://user-images.githubusercontent.com/41604114/131605648-5c417c85-2977-41b1-8aee-8530f177a91f.png">

Reply:
> 2. After installing and starting meshery this happen.
> <img alt="Screenshot 2021-09-01 at 8 28 24 AM" width="977" src="https://user-images.githubusercontent.com/41604114/131605210-f24288be-c284-43e6-ad08-d6844e89107a.png">
> 
> so, have to do `q`(quit) and `y` (yes) to get
> 
> <img alt="Screenshot 2021-09-01 at 8 39 54 AM" width="856" src="https://user-images.githubusercontent.com/41604114/131605503-ea559eb0-6733-4bef-80ec-9e9c5313d4e6.png">

@suhaniii we need to chase this issue down in the meshery.sh installer in `meshery/meshery`. Will you please create an issue and tag @hexxdump?

Reply:
@leecalcote  sure